src/components/layout/Footer/Footer.js

```diff
@@ -92,19 +92,19 @@ const Footer = ({ children }) => (
                 </li>
               </ul>
             </div>
-            <img src='./images/cards.png' alt='Supported credit cards' />
           </div>
+          <img src='./images/cards.png' alt='Supported credit cards' />
```

Comment:
Zmiana niby niewielka, ale wpływ na wyglad ma już spory, a na tabletach to się dobrze wyświetlało

Poniżej wkleiłem jak wygląda wersja na branchu `master`

![image](https://user-images.githubusercontent.com/7238770/95543975-3c1dec00-09fa-11eb-9f67-e3df316ac25f.png)

A tutaj jak wygląda Twoja wersja

![image](https://user-images.githubusercontent.com/7238770/95544017-5657ca00-09fa-11eb-9a62-a62239aeb6be.png)